pkg/apis/camel/v1/common_types.go

```diff
@@ -304,6 +304,8 @@ const (
 	CapabilityTracing = "tracing"
 	// CapabilityMaster defines the master capability
 	CapabilityMaster = "master"
+	// CapabilityResume defines the resume capability
+	CapabilityResume = "resume-kafka"
```

Comment:
I guess this need to be scoped as we did for the runtime i.e. CapabilityResumeKafka

Reply:
+1. I'll adjust. Thanks.